### ISSUE TITLE
Main window collapsed

### ISSUE THREAD

**Issue by worldendDominator** [briansd9/exile-diary#95]
Since a few days ago, I can't see the main window. The application starts and functions normally otherwise, it reacts to map-end triggers and shows overlay, but when I alt-tab to the window there's nothing at all even though the window is in focus. Holding alt-tab (so I get thumbnails of all windows) just shows a short top bar (with collapse/maximize/close buttons) and no window content. This persists when I close and restart the program or restart the PC.

**Comment by briansd9** [comment]
Can you open your settings file `%appdata%\exile-diary\settings.json` and search for "mainWindowBounds"? It has probably gotten set to some weird value... if so, try changing them to something like this (these are my settings, fullscreen at 1920x1080 resolution)

`"mainWindowBounds":{"x":-4,"y":-4,"width":1863,"height":1088}`

**Comment by worldendDominator** [comment]
Yes, somehow x/y both ended up at -32k. No idea why it happened.

However, setting them to sensible values doesn't seem to do anything. After starting up, the window is still missing, but the vaules I entered are not reset, so somehow the programs starts broken even with good settings.

**Comment by briansd9** [comment]
Hmm, maybe you edited it while the program was still running? It saves the window dimensions every so often, so your changes might have been overwritten.

Also, try deleting the whole `"mainWindowBounds":{...}`, this forces the window to be maximized on start.

**Comment by worldendDominator** [comment]
No, I made sure it was closed.

Anyway, removing that property fixed the problem. Thanks.
